pkg/GAPJulia/JuliaInterface/julia/ccalls.jl

```diff
@@ -2,12 +2,16 @@
 
 import Base: getproperty, propertynames
 
-function GET_FROM_GAP(ptr::Ptr{Cvoid})::Any
+function GAP_TO_JULIA(ptr::Ptr{Cvoid})::Any
```

Comment:
I don't know if this name is much better, as it is easy to confuse with `gap_to_julia`, even though `GAP_TO_JULIA` is merely a preparation for `gap_to_julia`.

Maybe something like `GAP_PTR_TO_JULIA`, to make even more clear what it does?

Reply:
I simply wanted something that is less confusing, and symmetric (so I can add JULIA_TO_GAP). How about `RAW_GAP_TO_JULIA` or `GAP_TO_JULIA_INTERN` ? Also note that I would not export this function. Can somehow make it "internal" to the module to prevent outsiders from calling it? Does Julia have naming conventions to mark "internal" functions?

Reply:
I am happy with `RAW`. Afaik, internal functions start with an `_`. But since you have exported and non-exported functions, private functions are more or less the not exported ones.